ietf/doc/views_conflict_review.py

```diff
@@ -409,13 +459,20 @@ def start_review_sanity_check(request, name):
 
     return doc_to_review
 
+
 def build_notify_addresses(doc_to_review):
```

Comment:
I'm thinking this would be more maintainable if it used the above utility suggestion and was implemented using set (still returning list)